apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -2121,6 +2152,17 @@ public static String buildEobId(ClaimTypeV2 claimType, String claimId) {
     return String.format("%s-%s", claimType.name().toLowerCase(), claimId);
   }
 
+  /**
+   * @param claimType the {@link ClaimTypeV2} to compute an {@link ExplanationOfBenefit#getId()} for
+   * @param claimId the <code>claimId</code> field value (e.g. from {@link
+   *     CarrierClaim#getClaimId()} to compute an {@link ExplanationOfBenefit#getId()} for
+   * @return the {@link ExplanationOfBenefit#getId()} value to use for the specified <code>claimId
+   *     </code> value
```

Comment:
Just a sidenote, but do we have a preference for inline tags `{@code xyz}` vs html tags `<code>xyz</code>`? I've seen both in BFD, and I know html tags can do a couple things that are difficult for inline tags, but personally I find inline tags a little more visually pleasing in an IDE.

Reply:
seems to be inconsistent. I think we should pick one and move to it and I like the inline tags too. I'm going to leave this one as-is to be consistent with the rest of the class for now. lets put it on the tech debt list

Reply:
fyi, added it.